src/components/recent-searches/RecentSearches.jsx

```diff
@@ -52,44 +52,50 @@ const RecentSearches = ({
                   setPrice010(
                     search.ticketPrice
                       ? search.ticketPrice.some(
-                          pr => pr.minPrice === 0 && pr.maxPrice === 10,
+                          (pr) => pr.minPrice === 0 && pr.maxPrice === 10,
                         )
                       : false,
                   )
 
                   setPrice1020(
                     search.ticketPrice
                       ? search.ticketPrice.some(
-                          pr => pr.minPrice === 10 && pr.maxPrice === 20,
+                          (pr) => pr.minPrice === 10 && pr.maxPrice === 20,
                         )
                       : false,
                   )
 
                   setPrice2040(
                     search.ticketPrice
                       ? search.ticketPrice.some(
-                          pr => pr.minPrice === 20 && pr.maxPrice === 40,
+                          (pr) => pr.minPrice === 20 && pr.maxPrice === 40,
                         )
                       : false,
                   )
 
                   setPrice4080(
                     search.ticketPrice
                       ? search.ticketPrice.some(
-                          pr => pr.minPrice === 40 && pr.maxPrice === 80,
+                          (pr) => pr.minPrice === 40 && pr.maxPrice === 80,
                         )
                       : false,
                   )
 
                   setPrice80(
                     search.ticketPrice
                       ? search.ticketPrice.some(
-                          pr => pr.minPrice === 80 && pr.maxPrice === 100000000,
+                          (pr) =>
+                            pr.minPrice === 80 && pr.maxPrice === 100000000,
                         )
                       : false,
                   )
 
-                  rccHistory.push(`/search${qs}`)
+                  // On user search, the components / page will not rereender, instead it will add on
+                  //  to the existing URL with what the user searches for, this however effects only the url
+                  // for user feedback
+                  window.history.pushState(null, null, `${qs}`)
```

Comment:
In addition to adding the search to the user history, can we dispatch a pageview to google analytics or some other way of tracking what the user searched for